docs/semgrep-code/java.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Are these Java docs supposed to be here?

Reply:
cry

Reply:
i branched out of the wrong place again